apps/frontend/src/pages/project/libs/components/contributors-list/libs/components/sync-time/sync-time.tsx

```diff
@@ -0,0 +1,61 @@
+import { Icon, Tooltip } from "~/libs/components/components.js";
+import { useEffect, useState } from "~/libs/hooks/hooks.js";
+
+import { OVERDUE_LIMIT } from "./libs/constants/constants.js";
+import { getStartOfDay, getSyncTime } from "./libs/helpers/helpers.js";
+import styles from "./styles.module.css";
+
+type Properties = {
+	lastActivityDate: null | string;
+	lastActivityUserName: null | string;
+};
+
+const SyncTime = ({
+	lastActivityDate,
+	lastActivityUserName,
+}: Properties): JSX.Element => {
+	const [isOverdue, setIsOverdue] = useState<boolean>(false);
+	const currentDate = getStartOfDay(new Date());
+
+	const lastActivity = lastActivityDate
+		? getStartOfDay(new Date(lastActivityDate))
+		: null;
+
+	const lastUpdateLabel = lastActivity
+		? getSyncTime(lastActivity, currentDate)
+		: null;
+
+	useEffect(() => {
+		if (lastUpdateLabel && lastUpdateLabel.hoursDifference !== undefined) {
```

Comment:
Try if it will work like this
```suggestion
		if (lastUpdateLabel?.hoursDifference !== undefined) {
```